test/executor.test.js

```diff
@@ -423,173 +473,207 @@ describe('executor', function() {
 
     it('should parse config variables in an object', function(done) {
       boot.execute(app, simpleMiddlewareConfig('routes',
         { info: { path: '${restApiRoot}' }}
-      ));
-
-      supertest(app).get('/').end(function(err, res) {
+      ), function(err) {
         if (err) return done(err);
-        expect(res.body.info).to.eql({
-          path: app.get('restApiRoot'),
+
+        supertest(app).get('/').end(function(err, res) {
+          if (err) return done(err);
+          expect(res.body.info).to.eql({
+            path: app.get('restApiRoot'),
+          });
+          done();
         });
-        done();
       });
     });
 
     it('should parse config variables in a nested object', function(done) {
       boot.execute(app, simpleMiddlewareConfig('routes',
         { nested: { info: { path: '${restApiRoot}' }}}
-      ));
-
-      supertest(app).get('/').end(function(err, res) {
+      ), function(err) {
         if (err) return done(err);
-        expect(res.body.nested).to.eql({
-          info: { path: app.get('restApiRoot') },
+
+        supertest(app).get('/').end(function(err, res) {
+          if (err) return done(err);
+          expect(res.body.nested).to.eql({
+            info: { path: app.get('restApiRoot') },
+          });
+          done();
         });
-        done();
       });
     });
 
     it('should parse config variables with null values', function(done) {
       boot.execute(app, simpleMiddlewareConfig('routes',
         { nested: { info: { path: '${restApiRoot}', some: null }}}
-      ));
-
-      supertest(app).get('/').end(function(err, res) {
+      ), function(err) {
         if (err) return done(err);
-        expect(res.body.nested).to.eql({
-          info: {
-            path: app.get('restApiRoot'),
-            some: null,
-          },
+
+        supertest(app).get('/').end(function(err, res) {
+          if (err) return done(err);
+          expect(res.body.nested).to.eql({
+            info: {
+              path: app.get('restApiRoot'),
+              some: null,
+            },
+          });
+          done();
         });
-        done();
       });
     });
 
     it('should not parse invalid config variables', function(done) {
-      var invalidDataTypes = [undefined, function() {}];
+      var invalidDataTypes = [undefined, function() {
+      }];
       async.each(invalidDataTypes, function(invalidDataType, cb) {
         var config = simpleMiddlewareConfig('routes', {
           path: invalidDataType,
         });
-        boot.execute(app, config);
-
-        supertest(app)
-          .get('/')
-          .end(function(err, res) {
-            expect(err).to.be.null();
-            expect(res.body.path).to.be.undefined();
-            cb();
-          });
+        boot.execute(app, config, function(err) {
+          if (err) return done(err);
+
+          supertest(app)
+            .get('/')
+            .end(function(err, res) {
+              expect(err).to.be.null;
```

Comment:
`expect(err).to.be.null()` was very intentional, see [Beware of libraries that assert on property access](https://github.com/moll/js-must#asserting-on-property-access). You should either revert back to chai version specified in the current package.json, or else setup chai with [dirty-chai](https://www.npmjs.com/package/dirty-chai) plugin.